TelCo.ColorCoder/IProgramHelpers.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+    public interface IProgramHelpers
```

Comment:
Interface name is IProgramHelper, this indicates a generic functionality. The interface name should be chosen to indicate  the contract that it is going to define. This is an improvement that you can make.